README.md

```diff
@@ -24,7 +24,7 @@ To write and publish content in Open Liberty Docs, you work with the following b
  
     `vNext` is source of each upcoming release. At release time, a cut is taken from this branch to make the release branch, which is then named `vX.0.0.x` according to the release version.
 
- Each branch is protected and all pull requests require review from a either a [doc member](https://github.com/orgs/OpenLiberty/teams/docs-members) or [doc maintainer](https://github.com/orgs/OpenLiberty/teams/docs-maintainers/members).
+ Each branch is protected and all pull requests require review from a either a [doc member](https://github.com/orgs/OpenLiberty/teams/docs-members) or [doc maintainer](https://github.com/orgs/OpenLiberty/teams/docs-maintainers/members). If you want to publish doc updates with the next OL release, your doc updates must be merged to the `vNext` branch by the Monday before the OL release. For example, if the OL release is Friday, 23 October 2020, then you must complete your doc updates and merge them to the `vNext` branch by Monday, 19 October 2020. The following sections provide further information about making doc updates and merging to the `vNext` branch. 
```

Comment:
- Should we spell out Open Liberty, since this is a public doc?
- Do we need to provide info that OL is on a 4 week release schedule, with the docs typically publishing on a Friday?
- "complete your doc updates and merge them to the `vNext` branch"- might need to mention that only docs maintainers can merge to vNext.